x-pack/plugins/monitoring/public/alerts/lib/get_alert_panels_by_node.tsx

```diff
@@ -38,18 +38,20 @@ export function getAlertPanelsByNode(
   } = {};
 
   for (const { states, rawAlert } of alerts) {
-    const { alertTypeId } = rawAlert;
+    const { id: alertId } = rawAlert;
     for (const alertState of states.filter(({ state: _state }) => stateFilter(_state))) {
       const { state } = alertState;
       statesByNodes[state.nodeId] = statesByNodes[state.nodeId] || [];
       statesByNodes[state.nodeId].push(alertState);
 
       alertsByNodes[state.nodeId] = alertsByNodes[state.nodeId] || {};
-      alertsByNodes[state.nodeId][alertTypeId] = alertsByNodes[alertState.state.nodeId][
-        alertTypeId
-      ] || { alert: rawAlert, states: [], count: 0 };
-      alertsByNodes[state.nodeId][alertTypeId].count++;
-      alertsByNodes[state.nodeId][alertTypeId].states.push(alertState);
+      alertsByNodes[state.nodeId][alertId] = alertsByNodes[alertState.state.nodeId][alertId] || {
+        alert: rawAlert,
+        states: [],
+        count: 0,
+      };
+      alertsByNodes[state.nodeId][alertId].count++;
+      alertsByNodes[state.nodeId][alertId].states.push(alertState);
```

Comment:
groups by alertId (unique UUID) instead of alertTypeId which can now happen twice due to support of multiple types of rules allowed.

Reply:
Perfect, thanks.